apps/openchallenges/mcp-server/build.gradle.kts

```diff
@@ -29,6 +29,15 @@ graalvmNative {
       buildArgs.add("--libc=musl")
       buildArgs.add("--no-fallback")
     }
+    named("test") {
```

Comment:
Enable the task `nativeTest`, which will not be used as part of the normal development workflow.